library/src/lib/calendar/calendar-header/calendar-header.component.ts

```diff
@@ -44,10 +44,12 @@ export class CalendarHeaderComponent implements OnInit {
         = new EventEmitter<void>();
 
     constructor(public calendarI18nLabels: CalendarI18nLabels,
-                public calendarI18n: CalendarI18n) {
+        public calendarI18n: CalendarI18n) {
     }
```

Comment:
Our style is more 
constructor(
    argument1,
    argument2
) {}

If there is more than 1 argument in constructor.